### ISSUE TITLE
form inputs not associated with labels

### ISSUE THREAD

**Issue by susanev** [chef/automate#3820]
## overview
fix some a11y issues across automate related to inputs not being associated with their labels. these were found using the [wave chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US).

## details
- [ ] projects dropdown across automate is not associated with its label
- [ ] policies dropdown in token modal is not associated with its label
